Switch to  postgres 16 as a minimum requirement (#5031)
theforeman/foreman-documentation: guides/common/modules/proc_installing-postgresql-as-an-external-database.adoc

The examples below assume PostgreSQL 14 as used on Ubuntu 22.04.
If you run {ProjectServer} on Debian 12, you have to adjust the paths of PostgreSQL config and data directories which depend on the PostgreSQL version.
endif::[]
ifdef::containerized[]

> {Project} supports PostgreSQL version 16, available in Enterprise Linux 10 repositories.

Reviewer: ```suggestion
{Project} supports PostgreSQL version 16 which is available in {EL} 10 repositories.
```

We have an attribute for this: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L82

Reviewer 2: ```suggestion
{Project} supports PostgreSQL version 16, which is available in {EL} 10 repositories.
```

endif::[]
ifndef::foreman-deb,containerized[]
{Project} supports PostgreSQL version 13.
endif::[]

> 

Reviewer: ```suggestion
```
